### PR TITLE
feat(csharp): adding csharp project

### PR DESCRIPTION
**Pull request by fernanduandrade**
Hi there!

This PR aims to add documentation and the base project for using ScyllaDB with Csharp, using [DataStax](https://github.com/datastax/csharp-driver/). I tried to make it as simple, following the others example already added so that everyone can learn a little more about and how to use it.

I hope it's ok! Please let me know if there is any modification need to be made!

Thanks to @DanielHe4rt  and @cherryramatisdev @lanjoni for inspiring me to contribute on this repository.

### CODE REVIEW

File: csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# ScyllaDB Cloud Media Player Metrics
+
+Project to store songs that you like to listen daily and keep track of them in a shape of a CLI!
+
+## Prerequisites
+
+* [Csharp]()
```
Comment (DanielHe4rt) [inline csharp/README.md:7]: Missing URL

File: docs/source/build-with-csharp.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+        
+        var session = cluster.Connect(keyspace);
+        
+        var result = session.Execute($"SELECT id, title, album, artist FROM {table}");
+
+        var rows = result
+                    .GetRows()
+                    .ToList();
+
+        foreach(var row in rows)
```
Comment (DanielHe4rt) [inline docs/source/build-with-csharp.md:274]: Isn't supposed to be using the "record" created on the line 252?
```suggestion
        foreach(Song row in rows)
```

File: csharp/Helper/DataBaseHelper.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+public static class DataBaseHelper
+{
+    public static bool RowSetHasResult(RowSet rowSet)
+    {
+        var rows = rowSet
+            .GetRows()
+            .ToList();
+
+        var hasResult = rows.Count > 0;
+        return hasResult;
```
Comment (DanielHe4rt) [inline csharp/Helper/DataBaseHelper.cs:14]: There's two better options IHMO, instead allocate a new variable to return the expression.

```suggestion
        var rows = rowSet
            .GetRows()
            .ToList();

        return rows.Count > 0;
```

```suggestion
        return rowSet
            .GetRows()
            .ToList()
            .Count > 0;
```

File: csharp/Cli.cs [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+    }
+
+    private async Task Stress()
+    {
+        var start = DateTime.Now;
+        Console.WriteLine("------------------------------------");
+        Console.WriteLine("Inserting 100.000 records into the database...");
+        Console.WriteLine(">    Starting...");
+        
+        var interation = 10000;
```
Comment (DanielHe4rt) [inline csharp/Cli.cs:141]: The counter doesn't matches with the description above. 

```suggestion
        var interation = 100000;
```

**Review by DanielHe4rt — CHANGES_REQUESTED** [review]
Hey @fernanduandrade! Thanks for the contribution, we really appreciate!

I did a few comments on things that you may missed, if you could please fix this we can follow with the merge! 

Cheers.

**Comment by fernanduandrade** [comment]
@DanielHe4rt I think it's all good now.

**Pull request merged**